elements/lisk-dpos/src/dpos.ts

```diff
@@ -148,6 +149,28 @@ export class Dpos {
 		return this.rounds.calcRoundStartHeight(activeRounds);
 	}
 
+	public async isStandByDelegate(
+		address: string,
+		height: number,
+		stateStore: StateStore,
+	): Promise<boolean> {
+		const forgersList = await getForgersList(stateStore);
+		const relevantRound = this.rounds.calcRound(height);
+		const foundForgerList = forgersList.find(
+			roundRecord => roundRecord.round === relevantRound,
+		);
+
+		if (!foundForgerList) {
```

Comment:
I think if no relevant forger list, it should an error (it is something wrong with the state